dev/com.ibm.ws.jaxws.cdi/bnd.bnd

```diff
@@ -40,3 +40,5 @@ Private-Package:\
 	com.ibm.ws.jaxws.clientcontainer;version=latest,\
 	com.ibm.ws.jaxws.common;version=latest,\
 	com.ibm.ws.managedobject;version=latest
+
+jakartaeeMe: true
```

Comment:
I see you added something in the direct properties for the transformer.  It seems you left something though in the file `javax.enterprise.context.Dependent`.  Not sure if it is needed at all, but it will need to be fixed up or removed.